content/doc/installation/with-docker.en-us.md

```diff
@@ -17,111 +17,179 @@ menu:
-```
-
-Now you should have a running instance of Gitea, to access the web UI just visit http://hostname:10080 in your favorite browser, if you want to clone repositories you can do it in the above case with `git clone ssh://git@hostname:10022/username/repo.git`.
-
-`hostname` should be replaced by ip adress of your gitea server.
-
-## Named Volumes 
-
-This guide will result in both the Gitea and PostgreSQL data stored in Docker named volumes. This makes for easy backup, restore and upgrades.
-
-### The Database
-
-First get the named volume ready:
-
-```
-$ docker volume create --name gitea-db-data
-```
-
-Now that we have a volume ready, we can pull and run the PostgreSQL image. This is also based upon Alpine Linux just like Gitea. This will mount the named volume at the correct location instead of a host dir.
-
-```
-$ docker pull postgres:alpine
-$ docker run -d --name gitea-db \
-    -e POSTGRES_PASSWORD=<PASSWORD> \
-    -v gitea-db-data:/var/lib/postgresql/data \
-    -p 5432:5432 \
-    postgres:alpine
-```
-
-Now the database is up and running, we need to configure it. Make sure you remember the password for when Gitea starts.
-
-```
-$ docker exec -it gitea-db psql -U postgres
-psql (9.6.1)
-Type "help" for help.
-
-postgres=# CREATE USER gitea WITH PASSWORD '<PASSWORD>';
-CREATE ROLE
-postgres=# CREATE DATABASE gitea OWNER gitea;
-CREATE DATABASE
-postgres=# \q
-$
-```
-
-### Gitea
-
-First, the named volume
-
-```
-$ docker volume create --name gitea-data
-```
-
-Now run (remove the dns entry if not needed):
-
-```
-$ docker run -d --name gitea \
-	--link gitea-db:gitea-db \
-	--dns 10.12.10.160 \
-	-p 11180:3000 \
-	-p 8322:22 \
-	-v gitea-data:/data \
-	gitea/gitea:latest
-```
-
-You should now have two Docker containers for Gitea and PostgreSQL plus two Docker named volumes and you should have a running instance of Gitea, to access the web UI just visit http://hostname:11180/install/ in your favorite browser.
-
-In the web UI set:
-- Database Type: PostgreSQL
-- Host: gitea-db:5432
-- User: gitea
-- Password: <PASSWORD>
-- ...
-- HTTP port: 11180
-- Application URL: http://hostname:11180
-
-Apply settings. You should be able to access to web UI now and create your first user.
+This reference setup guides you through the setup based on `docker-compose`, the installation of `docker-compose` is out of scope of this documentation. To install `docker-compose` follow the official [install instructions](https://docs.docker.com/compose/install/).
+
+## Basics
+
+The most simple setup just creates a volume and a network and starts the `gitea/gitea:latest` image as a service. Since there is no database available you can start it only with SQLite3. Create a directory like `gitea` and paste the following content into a file named `docker-compose.yml`.
+
+```yaml
+version: "2"
+
+networks:
+  gitea:
+    external: false
+
+services:
+  server:
+    image: gitea/gitea:latest
```

Comment:
I'd add `container_name`.

Reply:
That disables scaling entirely which is not recommended.

Reply:
Because I'm following a docker-compose standard, and all proper docker-compose documentations do not define a static name for that. Beside that if you are defining a static container name you can also use this container only once for the entire system.

Reply:
If you want to scale, you don't add it. But if not, you should add it. Please don't tell me you are following standards... This has nothing to do with standards. I think Gitea is not for scaling and as you are mapping ports to host, you can't scale it anyway!!!

Reply:
To be clear, I do not insist on adding it.